### PR TITLE
Ah schema edits

### PR DESCRIPTION
**Pull request by sriGanna**
This PR reflects changes to the Photo-audio-haptics-handler in response to changes to the semantic segmentation. This schema was tested on unicorn against most of the pictures of the image Demo Page as well as the following image:
https://en.wikipedia.org/wiki/File:Canyon_no_Lago_de_Furnas.jpg

Please ensure you've followed the checklist and provide all the required information *before* requesting a review.

## Required Information

- [ ] Reference the issue this is meant to fix or describe it if none exists.
- [ ] Full description of changes made.

## Pull Request Checklist

- [ ] Coding standards are followed (e.g., [PEP8](https://pep8.org/))
- [ ] An entry exists for the container in `docker-compose.yml` or `build.yml`
- [ ] The Docker image builds
- [ ] The container runs correctly when sent inputs with the changes
- [ ] No models or other large files are part of these commits
- [ ] A description of the tests already run as part of this PR is present
- [ ] CI is set up under `.github/workflows` or is not applicable

### CODE REVIEW

File: renderers/photoaudiohaptics.schema.json [hunk 1]
```diff
@@ -25,12 +25,16 @@
                 "maximum": 1
             }
         },
-        "contourPoints": {
+        "contours": {
```
Comment (JRegimbal) [inline renderers/photoaudiohaptics.schema.json:28]: To clarify, this is all the points with a label and not broken down by individual contours, right?

File: renderers/photoaudiohaptics.schema.json [hunk 2]
```diff
@@ -25,12 +25,16 @@
...
             "type": "array",
             "description": "An array containing an array of normalized coordinates representing a point on the segment or point on the bounding box.",
             "items": {
                 "$ref": "#/definitions/coordinates"
             }
+        },
+        
+        "entityType":{
```
Comment (JRegimbal) [inline renderers/photoaudiohaptics.schema.json:36]: Could you add more documentation for this? A description is a must, and if this only takes a few values then it should probably just be enumerated.

**Review by JRegimbal — CHANGES_REQUESTED** [review]

File: renderers/photoaudiohaptics.schema.json (hunk 2 above)
Comment (sriGanna) [reply renderers/photoaudiohaptics.schema.json:36]: Sorry, I forgot to add the description! I'll also add a max length for the string

File: renderers/photoaudiohaptics.schema.json (hunk 1 above)
Comment (sriGanna) [reply renderers/photoaudiohaptics.schema.json:28]: yes

File: renderers/photoaudiohaptics.schema.json [hunk 3]
```diff
@@ -25,12 +25,18 @@
...
+            "type": "string",
+            "description": "This describes whether the entity is a segment, object or static text",
```
Comment (JRegimbal) [inline renderers/photoaudiohaptics.schema.json:38]: Based on the PR with the handler code, it looks like there are only a few values for this. Could you change the type to an enumeration like below?
https://github.com/Shared-Reality-Lab/IMAGE-server/blob/ff3453c2d4974575f0f6b6681232184462c5b0e5/preprocessors/content-categoriser.schema.json#L8

**Review by JRegimbal — CHANGES_REQUESTED** [review]
One last change to this with just enumerating the entity types.

File: renderers/photoaudiohaptics.schema.json (hunk 3 above)
Comment (sriGanna) [reply renderers/photoaudiohaptics.schema.json:38]: yes, that would make sense!

**Review by JRegimbal — APPROVED** [review]

**Pull request merged**